pandora-client-web/src/components/wiki/pages/intro.tsx

```diff
@@ -0,0 +1,102 @@
+			</p>
+
+			<h4>Dynamically generated body model with many poses and free arm movement</h4>
+			<p>
+				In the pose tab the ability to freely move arms and legs is hidden away under a dev-section at the bottom of the screen.
+				You can expand it to show the according sliders.
+			</p>
+
+			<h4>Front and back character view</h4>
+			<p>
+				Can be toggled in the pose menu and is seen that way for everyone in the room.
+			</p>
+
+			<h4>A feature-rich room chat</h4>
+			<ul>
+				<li>Click a name in the chat to start whispering to that person</li>
+				<li>Right-clicking your own message enables you to edit or delete it for 10 mins after posting it</li>
+				<li>To get help on chat commands, start the command by typing the "/" character which will start showing a list of available commands</li>
+				<li>You can write Out-Of-Character (OOC) messages by prefixing them with "(("</li>
+				<li>You can enclose text with underscores to style it, such as _<i>italic</i>_ and __<b>bold</b>__</li>
+			</ul>
+
+			<h4>Free character placement and movement inside rooms</h4>
+			<p>
+				You can freely move your character inside a room by dragging them by the character name below.
+				Room admins can also move other characters this way.
+			</p>
+
+			<h4>Room-level furniture and devices that can be placed freely and that persist with the life-time of the room</h4>
+			<p>
+				Room devices first have to be deployed from the room inventory to the room in the item menu.
+				Then, the room devices can be freely moved/positioned in the room by dragging them by the icon below.
+				The icons below room devices can be toggled to be hidden under the "Controls"-tab.
+			</p>
+
+			<h4>Stable code base</h4>
+			<p>
+				Pandora aims for a stable experience without random disconnects. But even if a short disconnect happens, the character will not be shown as disconnected for some time and you will still receive all the missed chat messages on reconnect, not loosing anything.
+			</p>
+
+			<h4>Actually "secure" gag talk & locks</h4>
+			<p>
+				Pandora validates and performs all character interactions on the server,
+				preventing the creation of scripts/mods that do undesired actions, such as anti-garble or unauthorized removal of (locked) restraints.<br />
+				This ensures that you can depend on what others see,
+				resulting in everyone having the same experience.
+			</p>
+
+			<h4>No safeword feature, but a safemode that makes it harder to be misused</h4>
+			<p>
+				The safemode feature can be found by clicking on your character name in the top left of the screen and then entering safemode in the menu.
+			</p>
+
+			<hr />
+
+			That is not all of course! There are many features planned for the future:
+			<ul>
+				<li>Item templates that store your favorite asset configurations per asset, including things like color, custom name,</li>
+				description/lore about the item, etc.
+				<li>Wardrobe outfit saving feature as well as import/export</li>
```

Comment:
maybe: `Feature for saving wardrobe outfits and supporting import/export functionality`